internal/internal_task_handlers_test.go

```diff
@@ -1214,6 +1215,97 @@ func (t *TaskHandlersTestSuite) TestWorkflowTask_PageToken() {
 	t.NotNil(response)
 }
 
+func (t *TaskHandlersTestSuite) TestLocalActivityRetry_Workflow() {
```

Comment:
Should we add a test that validates scenario mentioned in https://github.com/uber-go/cadence-client/issues/915?

Reply:
This one doesn't - but the existing one directly after it does, so I think the path is covered.

Reply:
Yeah, I see, there is `TestLocalActivityRetry_WorkflowTaskHeartbeatFail`. Looks good to me then.